samples/bluetooth/peripheral_mds/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Heap memory is required for the memfault_demo_cli.c
```

Comment:
It is actually no true because 4k of memory is needed by OpenAMP so maybe it should be 4k plus something

Reply:
Good point.